typedb-src/modules/ROOT/pages/quickstart-guide.adoc

```diff
@@ -213,29 +213,26 @@ For more TypeQL query examples, check the xref:typeql:ROOT:overview.adoc[TypeQL]
 [#_get_query__retrieve_all_data]
 === Get all data
 
-The following query retrieves all data from a database.
-
 [,typeql]
 ----
 match $t isa thing;
 ----
 
+Retrieves all data from a database.
```

Comment:
```suggestion
Retrieve all data from a database:
```
The typical voice for sentences like this in English is imperative. Please change throughout if you've chosen to shorten a sentence in a parallel way.

I also prefer a colon (':') as an introduction, but this is a stylistic opinion.